Doc/whatsnew/3.11.rst

```diff
@@ -471,17 +471,24 @@ Other Language Changes
   there is a mixture of strings and bytes keys.
   (Contributed by Thomas Grainger in :gh:`91181`.)
 
+
+.. _whatsnew311-other-implementation-changes:
+
 Other CPython Implementation Changes
 ====================================
 
-* Special methods :meth:`complex.__complex__` and :meth:`bytes.__bytes__` are implemented to
-  support :class:`typing.SupportsComplex` and :class:`typing.SupportsBytes` protocols.
+* The special methods :meth:`~object.__complex__` for :class:`complex`
+  and :meth:`~object.__bytes__` for :class:`bytes` are implemented to support
+  the :class:`typing.SupportsComplex` and :class:`typing.SupportsBytes` protocols.
   (Contributed by Mark Dickinson and Dong-hee Na in :issue:`24234`.)
 
-* ``siphash13`` is added as a new internal hashing algorithms. It has similar security
-  properties as ``siphash24`` but it is slightly faster for long inputs. ``str``, ``bytes``,
-  and some other types now use it as default algorithm for :func:`hash`. :pep:`552`
-  hash-based pyc files now use ``siphash13``, too.
+* ``siphash13`` is added as a new internal hashing algorithm.
+  It has similar security properties as ``siphash24``,
+  but it is slightly faster for long inputs.
+  :class:`str`, :class:`bytes`, and some other types
+  now use it as the default algorithm for :func:`hash`.
+  :pep:`552` :ref:`hash-based .pyc files <pyc-invalidation>`
+  now use ``siphash13``, too.
```

Comment:
```suggestion
  now use ``siphash13`` too.
```
Is the comma necessary? :thinking:

Reply:
It was in the original and reflect a pause when reading it out loud, so I didn't touch it, but it isn't syntactically or semantically necessary, and yeah I don't think its vital to retain it.